.devcontainer/devcontainer.json

```diff
@@ -1,36 +1,37 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/python
 {
-    "name": "Functions Quickstarts Codespace",
-    "dockerFile": "Dockerfile",
-    "features": {
-      "azure-cli": "latest"
-    },
-    "customizations": {
-      "vscode": {
-        "extensions": [
-          "ms-azuretools.vscode-bicep",
-          "ms-azuretools.vscode-docker",
-          "ms-azuretools.vscode-azurefunctions",
-          "GitHub.copilot",
-          "humao.rest-client"
-        ]
-      }
-    },
-    "mounts": [
-      // Mount docker-in-docker library volume
-      "source=codespaces-linux-var-lib-docker,target=/var/lib/docker,type=volume"
-    ],
-    // Always run image-defined docker-init.sh to enable docker-in-docker
-    "overrideCommand": false,
-    "remoteUser": "codespace",
-    "runArgs": [
-      // Enable ptrace-based debugging for Go in container
-      "--cap-add=SYS_PTRACE",
-      "--security-opt",
-      "seccomp=unconfined",
-  
-      // Enable docker-in-docker configuration
-      "--init",
-      "--privileged"
-    ]
-  }
-  
\ No newline at end of file
+	"name": "Functions Quickstarts Codespace - Python, aarch64 ready",
+	"dockerFile": "Dockerfile",
+	"features": {
+		"azure-cli": "latest"
+	},
+	"forwardPorts": [7071],
+	"portsAttributes": {
+		"7071": {
```

Comment:
optional: should we mention why this port is being used in the first-run-notice.txt section?